views/projects.tpl

```diff
@@ -135,6 +135,7 @@
             <thead>
                 <tr>
                     <!-- The "Professor - Lab" column is displayed only on the "all projects" page -->
+                    <th>Category</th>
```

Comment:
I'd invert these two lines, to keep the comment close to the line to which it refers. :)